src/treetensornetworks/solvers/tree_sweeping.jl

```diff
@@ -1,76 +1,71 @@
-# 
-# Sweep step
-# 
+direction(step_number) = isodd(step_number) ? Base.Forward : Base.Reverse
 
```

Comment:
`make_region` below is now a bit complicated, but it is the unification of the four previous functions one_site_region, two_site_region, tdvp_one_site_region, tdvp_two_site_region.